### PR TITLE
[CS2113-T10-2] Location Buddy

### PR DESCRIPTION
**Pull request by tehtea**
An app that helps you finds your path around the school

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Class Diagram2.svg)
```
Comment (JethroPhuah) [inline docs/DeveloperGuide.md]: Diagram is not working. Unable to load.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+
+### Sequence digram for v2.0
+### Sequence digram for listAllLocations
+![Sequence Digram of listAllLocations](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Sequence%20Digram.JPG)
```
Comment (ManikaHennedige) [inline docs/DeveloperGuide.md]: Good morning, unfortunately it seems like the links to all of the diagrams aren't functional - maybe can try a relative path import instead?
image

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -2,7 +2,14 @@
...
+### Sequence digram for findNearest
+![Sequence Digram of findNearest](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/findNearest.png)
```
Comment (ManikaHennedige) [inline docs/DeveloperGuide.md]: I think an accompanying detailed description of the sequence diagram would be appropriate here - I'm not sure if the description is included in the .jpg file, though, but some explanation of the contents of the diagram would be nice!
- another minor issue: diagram is spelt wrongly
image

**Comment by JethroPhuah** [comment]
Left a Review

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -2,33 +2,82 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](assets/classdiagram.svg)
```
Comment (kangxinwang) [inline docs/DeveloperGuide.md]: maybe you can leave out less important details and make the class diagram a bit bigger so it is more readable?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -2,33 +2,82 @@
...
+We have 4 packages in total: Control, Entity, Seedu.Duke(boundary) and exception. 
+Control classes deal with the database of entity objects directly, return the wanted result after processing the database.  
+Entity classes include facilities that are available to be navigated to.
+Boundary classes take in user input and process the user input to pass to control class.
+Exception handles common exceptions such as invalid input exception etc to make sure the system runs smoothly.
+
+
+### Sequence digram for v2.0
+### Sequence digram for listAllLocations
+![Sequence Digram of listAllLocations](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Sequence%20Digram.JPG)
```
Comment (Zufiqqar) [inline docs/DeveloperGuide.md]: The Image for Class Diagram is working. However, for the Sequence diagram Images, there are some bugs.

Maybe can fix it by using URL: docs/assets/Sequence%20Digram.JPG

Comment (kangxinwang) [inline docs/DeveloperGuide.md]: seems like you could include your user stories here and likewise i think you did not link your image properly so it is not shown on the developer's guide

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -2,33 +2,82 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](assets/classdiagram.svg)
+
+We have 4 packages in total: Control, Entity, Seedu.Duke(boundary) and exception.
```
Comment (Zufiqqar) [inline docs/DeveloperGuide.md]: Good brief description of the Class diagram. Maybe you can do the same thing for the Sequence Diagram.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -2,33 +2,82 @@
...
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+###Version 2.0
+
+1. Find nearby libraries
```
Comment (Zufiqqar) [inline docs/DeveloperGuide.md]: image

I think the numbering does not reflects what you wanted. Could be some bugs

**Pull request closed without merging**